core/app/models/spree/order.rb

```diff
@@ -583,8 +583,89 @@ def fully_discounted?
     end
     alias_method :fully_discounted, :fully_discounted?
 
+    def add_store_credit_payments
```

Comment:
Assignment Branch Condition size for add_store_credit_payments is too high. [21.21/15]